fairscale/experimental/nn/sync_batchnorm.py

```diff
@@ -62,6 +40,87 @@ def _forward(
     _forward = torch.jit.script(_forward)  # type: ignore
 
 
+class _SyncBatchNormFunction(torch.autograd.Function):
```

Comment:
on line 39, I think you can use torch_version() function

Reply:
we can move that function our to testing.py as well since we do use it in non testing code.